masterlist.yaml

```
@@ -3129,6 +3129,12 @@ plugins:
       - crc: 0x074D8C60
         util: 'SSEEdit v4.0.3'
 
+  - name: 'A Clear Map of Skyrim.esp'
+    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/56367/' ]
+    group: *worldSettingsGroup
+    after:
+      - 'Atlas Map Markers.esp'
```

This list can be condensed. Otherwise, looks good to me.

```suggestion
    after: [ 'Atlas Map Markers.esp' ]
```